extensions/wikia/FlowTracking/scripts/createNewPageFlows.js

```diff
@@ -8,25 +8,43 @@ define(
 			articleId = mw.config.get('wgArticleId');
 
 		function trackOnEditPageLoad(editor) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AFunctionComplexity)